src/plugins/intel_gpu/src/graph/graph_optimizer/remove_redundant_reorders.cpp

```diff
@@ -554,6 +555,47 @@ void remove_redundant_reorders::run(program& p) {
         }
     }
 
+    // Remove reorder in front of MVN
+    // when the output format of reorder is bfyx and
+    // the output format of previous node of reorder is supported by mvn kernel.
+    itr = p.get_processing_order().begin();
+    while (itr != p.get_processing_order().end()) {
```

Comment:
Do we really need separate graph traverse here? Can't this reorder be removed if you just extend `can_fuse_reorder_to_prev` or `can_fuse_reorder` methods?

Reply:
That was considered first. But I thought removing reorder node from there could affect other nodes. I will review it again.

Reply:
I agree with @vladimir-paramuzov  I think `can_fuse_reorder` would be a better place..